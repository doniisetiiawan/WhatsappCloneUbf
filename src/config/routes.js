import Home from '../screens/Home';
import ChatScreen from '../screens/ChatScreen';

const routes = {
  home: { name: 'home', screen: Home },
  chat: { name: 'chat', screen: ChatScreen },
};

export default routes;
