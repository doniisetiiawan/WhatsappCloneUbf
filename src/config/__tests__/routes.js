import React from 'react';
import routes from '../routes';

it('has all needed routes', () => {
  expect(routes).toMatchSnapshot();
});
