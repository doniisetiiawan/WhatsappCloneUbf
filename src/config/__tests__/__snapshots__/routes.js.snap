// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has all needed routes 1`] = `
Object {
  "chat": Object {
    "name": "chat",
    "screen": [Function],
  },
  "home": Object {
    "name": "home",
    "screen": [Function],
  },
}
`;
